shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthenticationAttempts.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.di.authentication.shared.entity;
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbAttribute;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+public class AuthenticationAttempts {
+
+    private String attemptIdentifier;
+    private String email;
+    private String journeyType;
+    private String authenticationType;
+    private String code;
+    private Integer count;
+    private long timeToLive;
+    private String created;
+    private String updated;
+
+    @DynamoDbPartitionKey
+    @DynamoDbAttribute("AttemptIdentifier")
+    public String getAttemptIdentifier() {
+        return attemptIdentifier;
+    }
+
+    public void setAttemptIdentifier(String attemptIdentifier) {
+        this.attemptIdentifier = attemptIdentifier;
+    }
+
+    public AuthenticationAttempts withAttemptIdentifier(String attemptIdentifier) {
+        this.attemptIdentifier = attemptIdentifier;
+        return this;
+    }
```

Comment:
Will `attemptIdentifier` contain `internalSubjectId`?

Reply:
@dbes-gds can you please have a look over this?

Reply:
I think we should merge this but there may be an enhancement around this that we can pickup next week to make it more explicit what the identifier contains.